src/main/java/de/dhbw/karlsruhe/students/mailflow/core/application/auth/LoginUseCase.java

```diff
@@ -0,0 +1,9 @@
+package de.dhbw.karlsruhe.students.mailflow.core.application.auth;
+
+import de.dhbw.karlsruhe.students.mailflow.core.domain.auth.AuthorizationException;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.auth.LoadingUsersException;
+
+public interface LoginUseCase {
+
+  void login(String email, String password) throws AuthorizationException, LoadingUsersException;
```

Comment:
I thougt we were using  a Addres object instead ob a string for the email or am I wrong?

Reply:
Yes this is still the case. I can change it if you want. You guys are probably right that we can still provide the Address object. I thought it might be better decoupled if we do Address.from() inside the service instead of the CLI but the more I think about it, i see that it actually does not matter